seqr/views/apis/igv_api.py

```diff
@@ -126,9 +131,39 @@ def fetch_igv_track(request, project_guid, igv_track_path):
     if igv_track_path.endswith('.bam.bai') and not does_file_exist(igv_track_path, user=request.user):
         igv_track_path = igv_track_path.replace('.bam.bai', '.bai')
 
+    if is_google_bucket_file_path(igv_track_path):
+        return _stream_gs(request, igv_track_path)
+
     return _stream_file(request, igv_track_path)
 
 
+def _stream_gs(request, gs_path):
+    headers = {'Authorization': 'Bearer {}'.format(_get_access_token(request.user))}
+    range_header = request.META.get('HTTP_RANGE')
+    if range_header:
+        headers['Range'] = range_header
+    if gs_path.startswith('gs://fc-secure'):
+        headers['x-goog-user-project'] = 'anvil-datastorage'
```

Comment:
Can we make this a helper method in file_utils as well, since we do the same check there? I just feel like theres a chance that someday we would need to change this project ID or the logic to check for it and I would rather only have one place to go

Reply:
Maybe a helpoer like `get_google_project_id` that takes the gs path and returns either `'anvil-datastorage'` or None, and then here we would use it
```
google_project = get_google_project(gs_pth)
if google_project:
     headers['x-goog-user-project'] = google_project
```

Reply:
It seems that we could split the Google Storage codes into another file. Such as gs_utils.py

Reply:
> Maybe a helpoer like `get_google_project` that takes the gs path and returns either `'anvil-datastorage'` or None, and then here we would use it

Oh... I see what you mean.